13/umbraco-ui-builder/collections/retrieve-child-collections.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Configuring **one-to-many** relationships in Umbraco UI Builder, the backoffice UI builder for Umbraco.
+---
+
+# Retrieve Child Collections
+
+In a **one-to-many** relationship context, an entity from a parent collection can be associated with multiple entities from a child collection. In this context, you might need 
+to retrieve the child collection entities, without fetching the details of the parent.
+
+## Models Representation
+
+The Poco models would look like this:
+
+```csharp
+[TableName("Person")]
+[PrimaryKey("Id")]
+public class Person
+{
+    [PrimaryKeyColumn]
+    public int Id { get; set; }
+
+    public string Name { get; set; }
+
+    public string Email { get; set; }
+}
+```
+
+```csharp
+[TableName("ChildPerson")]
```

Comment:
Can we come up with a more real-world example than "child person"?

Reply:
Switched to a model called `StudentProjects`.